src/main/java/com/ssafy/sandbox/paging/controller/ArticleController.java

```diff
@@ -0,0 +1,64 @@
+package com.ssafy.sandbox.paging.controller;
+
+import com.ssafy.sandbox.paging.domain.Article;
+import com.ssafy.sandbox.paging.dto.ArticleResponse;
+import com.ssafy.sandbox.paging.dto.ArticlesByCursorResponse;
+import com.ssafy.sandbox.paging.dto.ArticlesByOffsetResponse;
+import com.ssafy.sandbox.paging.service.ArticleService;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/articles/paging")
+public class ArticleController {
+
+    private final ArticleService articleService;
+    private static final String DEFAULT_PAGE_NO = "1";
+    private static final String DEFAULT_PAGE_SIZE = "10";
+    private static final String DEFAULT_CURSOR_ID = "0";
+
+    @GetMapping("/offset")
+    public ResponseEntity<ArticlesByOffsetResponse> findArticleByOffset(
+            @RequestParam(defaultValue = DEFAULT_PAGE_NO) Integer page,
+            @RequestParam(defaultValue = DEFAULT_PAGE_SIZE) Integer size) {
+        Pageable pageable = PageRequest.of(page, size);
```

Comment:
```suggestion
        Pageable pageable = PageRequest.of(page, size, Sort.by(Sort.Direction.DESC, "createdAt"));
```
offset 쿼리에는 order by가 적용되지 않고 있는데, 이 경우에 limit을 수행하기 위해서 테이블 풀 스캔이 일어난다고 합니다. 현재는 데이터가 몇 백개라 문제가 되지 않지만 고려해볼 사안인 것 같아서 제안드립니다.
참고 : [mysql limit 동작 by 민서오빠 공유](https://jeong-pro.tistory.com/244)

Reply:
정렬 기준이 되는 컬럼에 인덱스가 걸려있어야 풀 스캔이 일어나지 않는 것으로 알고 있습니닷😊